ViaEventAssociation.Core.Domain/Aggregates/ViaEvents/ValueObjects/Description.cs

```diff
@@ -19,23 +19,20 @@ public static Result<Description> Create(string descriptionText)
             return Result<Description>.Failure(validationResult.Errors.ToArray());
         }
 
-        return Result<Description>.Success(new Description(descriptionText));
+        var finalDescriptionText = string.IsNullOrEmpty(descriptionText) ? "No description provided." : descriptionText;
```

Comment:
description should be seted to "" empty string if was null  {US2 S1}.

Reply:
done